src/services/etherspot.js

```diff
@@ -623,6 +625,24 @@ export class EtherspotService {
     }
   }
 
+  async fetchCollectibles(walletAddress: string): Promise<?(OpenSeaAssets)> {
```

Comment:
Can we rename Collectibles to NFTs? We did this in UI in most of cases, it would be great to have it renamed now in code too.
```suggestion
  async fetchCollectibles(walletAddress: string): Promise<?(OpenSeaAssets)> {
```

Reply:
okay will do that